src/hooks/useHandleTransfer.tsx

```diff
@@ -900,6 +937,7 @@ async function sei(
       "Wormhole - Complete Transfer"
     );
 
+    onStart?.();
```

Comment:
I think this line should be after `const tx` var, because you can include the tx id in this event https://github.com/wormhole-foundation/wormhole-connect/blob/release/wormhole-connect/src/views/Bridge/Send.tsx#L170